### DISCUSSION TITLE
[CR] autostart option

### DISCUSSION THREAD

**Discussion by Testato** [snappysnappydog/Simple-IP-Config#162] [Ideas]
I know that it's possible manually add the program to the Startup folder, but it would be better have a new option in the settings that when flagged  autostart the program when windows start

**Comment by KurtisLiggett** [comment]
Good idea, and doesn't seem like it would be too hard to implement.

**Comment by blackcrack** [comment]
@KurtisLiggett 
not hard, knowing is Jedi-force.. and knowing is might .. 
and if you know, how you be in able for adding a registrie string with the path of the own file....
....**then be you mighty** 

make a runtime "addautost(){}" 
you have add, like i guess the possible to make with the own exe-file different things, like the own path and the own-binary.. 
let check, if exist the registry-string from the path/binary existent or not,   
if existent you become a =1 and not =0 
make a checkbox with text behind "Autostart", at "=1" let it checked ..
if not,.. unchecked..  
by Start of the Program.. let run "addautost" and if is checked, then all okey,  
if will be checked by mouse, then add   the regestrystring..  
you can also use "reg add ~" the internal command of any nt system...  
and behind can you run again "addautost" for let check if it was successful and  
let check the checkbox behind the   successful action ..  
  
with "REG" can you make many type it in the "cmd", "reg /?" and check out, where is the right place.. an hint, there exist also an runonce ;) at the side of mashin and user .. play a bit around with the regestry @KurtisLiggett ,
if you see and understand, makes a bit fun to work with it.. :) the same with Workgroup (  if you remember to the issue, with "net config")  
  
best :)
